package.json

```diff
@@ -18,14 +22,15 @@
     "build:docker": "docker-compose -f docker-compose.yml up -d --build",
     "test": "BRANCH=local AWS_XRAY_CONTEXT_MISSING=LOG_ERROR jest --testMatch=\"**/*.unitTest.ts\" --coverage",
     "test-i": "BRANCH=local jest --testMatch=\"**/*.intTest.ts\" --runInBand",
+    "test-local-i": "USE_CONTAINERIZED_DATABASE=1 BRANCH=local jest --testMatch=\"**/*.intTest.ts\" --runInBand",
     "tslint": "tslint src/**/*.ts tests/**/*.ts -q",
     "prepush": "npm test && npm run tslint",
     "sonar-scanner": "AWS_XRAY_CONTEXT_MISSING=LOG_ERROR BRANCH=local npm run tslint && AWS_XRAY_CONTEXT_MISSING=LOG_ERROR npm run coverage && sonar-scanner",
     "coverage": "BRANCH=local SLS_DEBUG=* jest --testMatch=\"**/*.*Test.ts\" --coverage",
     "audit-dep": "npm audit --json | node_modules/.bin/audit-filter --nsp-config=.nsprc --audit=-",
```

Comment:
if we add the .npmrc file we can remove this script and replace it with that `"audit": "npm audit"` which is ran in the pipeline

Reply:
@githanium As far as I'm aware the current structure works for this project - have I misunderstood?

Reply:
@JonLazariniDD's approach allows developers to customise the behaviour of the audit process in  the repo itself. This approach will give you the ability to set the audit level as PO wish in the repo. The pipeline instead just run npm audit without much config to be modified @PaulBenn-BJSS

Reply:
resolving this - Jon has double-checked current config runs the proper auditing process in the pipeline